sample.xo-install.cfg

```diff
@@ -121,6 +121,13 @@ PRESERVE="3"
 # default: true
 #INSTALL_REPOS="true"
 
+# If set to true, this will install a small third-party copr repository necessary to install the libvhdi and libvhdi-tools rpms as-well-as the libvhdi and libvhdi-tools rpms.
+# If set to false, this repository and the corresponding libvhdi and libvhdi-tools rpms will not be installed.  As as result, file-level restore from delta backups within XOA will NOT work.
+# This is specific to Redhat/RockyLinux/AlmaLinux installs ONLY.
+# options: true/false
+# default: false
+#INSTALL_EL_LIBVHDI="false"
```

Comment:
Add the default value to the beginning of `xo-install.sh` too where other defaults are.